changelog.md

```diff
@@ -19,9 +46,56 @@ If the limit of concurrent requests is reached, a HTTP 503 Service Temporarily U
 which means that this operation can be retried.
 You can turn on this feature by setting the parameter `--http_max_concurrent_requests`.
 
+#### Serialize all concurrent change requests
+ 
+In former versions of Marathon it was possible that multiple concurrent changes of the same application definition
+would lead to an inconsistent state of that application. All changes of groups or application definitions now
+get serialized, so the resulting state is always consistent.
+
+#### Enhance the reported metrics 
+
+The metrics endpoint (`/metrics`) gives valuable insight into the system behavior.
+We now have meters for all service objects as well as the number applications and groups in the system.
+
+#### Decouple API (un)marshalling types from internal models
+
+With this change we can control API changes and are free to change internal models.
+This will change nothing in the current API, but will help a lot while we create the v3 API. 
+
+#### Added integration tests for all API endpoints
+
+We now have an integration test suite, that covers all endpoints with almost all possible parameter combinations.
+This test suite runs with every commit on every pull request.
+This test suite is our safety net to ensure compatibility while introducing new functionality in future versions.
+
+#### Code coverage reporting using coveralls
+
+See our github page https://github.com/mesosphere/marathon or go directly to  
+https://coveralls.io/github/mesosphere/marathon to see our code coverage reporting.
+With every Pull Request we try to make sure, this coverage will be increased.  
+
+#### Improved the release process of Marathon itself
+
+Several changes are made to the sbt build file, but also to the infrastructure that is performing the release.
+We now have a more reliable way of pushing distributions.
+
+
 ### Fixed Bugs
 
+- #1710 - Too many simultaneous requests will kill Marathon 
+- #1709 - Concurrent changes to the same AppDefinition will lead to inconsistent state
+- #1669 - Inconsistent content-type and response from REST API 
+- #1660 - The GUI doesn't allow creating apps with 0 instances, but it is possible through the v2 API 
 - #1647 - Keep the rest-api return format consistent when request headers without "Accept: application/json"
+- #1397 - Kill tasks with scaling does not update group
+- #1654 - Marathon 0.8.1 - API - /v2/tasks 
+- #555  - Constraints not satisfied when scaling down
+- #650  - deployment_success/failed should contain deployment info  
+- #1316 - Invalidate framework ID if registration triggers an error  
+
```

Comment:
After #1807 gets merged, add:
- #1254 - Add flag to define Marathon PORT prefix